src/amber/cli/templates/auth.cr

```diff
@@ -14,8 +14,8 @@ module Amber::CLI
     @primary_key : String
 
     def initialize(@name, fields)
-      @database = database
-      @language = language
+      @database = Amber::CLI::Config.database
+      @language = Amber::CLI::Config.language
```

Comment:
Do we want to fully namespace this, or have an `include Config` at the top of this class?

Like this: https://github.com/amberframework/amber/blob/master/src/amber/cli/templates/auth.cr#L7